routes/authRouter.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const authController = require("../controllers/authController");
+const errorHandler = require("../middlewares/errorHandler");
+
+const authRouter = express.Router();
+
+authRouter.get("/logIn", errorHandler(authController.logIn));
```

Comment:
```suggestion
authRouter.get("/kakao-login", errorHandler(authController.logIn));
```
확장성을 고려해서라도 정확하게 작성해주세요!